.github/workflows/check-catalyst.yaml

```diff
@@ -146,6 +146,63 @@ jobs:
         COMPILER_LAUNCHER="" \
         make llvm
 
+  llvm-sanitize:
+    name: LLVM Build (with ASAN and USAN)
+    runs-on: ubuntu-latest
```

Comment:
Did you decide to build both versions then and not build mlir-hlo with the sanitizer? :)

Reply:
Yep, it's not just mlir-hlo anymore. I didn't want us dealing with mlir-hlo and Enzyme issues with the sanitizer.

Reply:
Makes sense 👍